endpoints.tf

```diff
@@ -67,7 +67,7 @@ data "template_file" "local_metastores_yaml" {
     metastore_host   = "${lookup(var.local_metastores[count.index],"host")}"
     metastore_port   = "${lookup(var.local_metastores[count.index],"port")}"
     mapped_databases = "${ lookup(var.local_metastores[count.index],"mapped-databases","") }"
-    whitelist        = "${ lookup(var.local_metastores[count.index],"whitelist","") }"
+    writable_whitelist        = "${ lookup(var.local_metastores[count.index],"writable-whitelist","") }"
```

Comment:
Can you run `terraform fmt` please?

Reply:
we can run later or in another PR, I wan't to minimize changes while we have multiple PRs pending to avoid conflicts.